mbid_mapping/mapping/mb_metadata_cache.py

```diff
@@ -375,6 +383,10 @@ def get_metadata_cache_query(self, with_values=False):
                               FROM musicbrainz.recording r
                               JOIN musicbrainz.artist_credit ac
                                 ON r.artist_credit = ac.id
+                         LEFT JOIN recording_first_release_date rfdr
```

Comment:
If we have access to this recording_first_release_date, we could send it to the front-end and display it.